planner/cascades/testdata/transformation_rules_suite_in.json

```diff
@@ -40,6 +40,7 @@
       "select a, b, c from t t1 where t1.a in (select a from (select t2.a as a, t1.b as b from t t2 where t2.b > t1.b) x order by b limit 1)",
       "select a, b from (select @i as a, @i := @i+1 as b from t) t order by a desc limit 1",
       "(select a from t) union all (select b from t) order by a limit 2;",
+      "(select a from t) union all (select b from t) limit 2;",
```

Comment:
Also add a test case for limit with offset.

Reply:
Sure, will do.